gtfs/spec/en/reference.md

```diff
@@ -766,7 +767,7 @@ Defines the booking rules for rider-requested services
 
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-| `booking_rule_id` | ID | **Required** | Identifies the rule. |
+| `booking_rule_id` | ID | **Required** | Identifies a rule or set of rules. |
```

Comment:
```suggestion
| `booking_rule_id` | Unique ID | **Required** | Identifies a rule. |
```
Sorry for the error and confusion here. The `booking_rule_id` as the primary key itself, is a unique ID